core/orchestrator.go

```diff
@@ -233,7 +233,7 @@ func (n *LivepeerNode) transcodeAndCacheSeg(config transcodeConfig, ss *SignedSe
 	transcodeStart := time.Now().UTC()
 	// Ensure length matches expectations. 4 second + 25% wiggle factor, 60fps
 	if err := ffmpeg.CheckMediaLen(fname, 4*1.25*1000, 60*4*1.25); err != nil {
-		glog.Errorf("Media length check failed: %v", err)
+		glog.Errorf("Keyframe interval possibly > 4 seconds: %v", err)
```

Comment:
Note that this doesn't actually return the modified message to the broadcaster, it only prints it out on the orchestrator.

If you do it this way, it should look something like
```
if err.Error() == "MediaStats Failed" {
  err = fmt.Errorf("Keyframe interval possibly > 4 seconds")
}
```

Reply:
I see what you mean. The message needs to go to the broadcaster, and not just be printed out on the orchestrator. So it would have to be returned by this function and printed out in the broadcaster.